library/bencode.c

```diff
@@ -30,7 +30,7 @@
 #include <stdint.h>
 #include <sys/stat.h>
 
-#include "bencode.h"
+#include "../include/bencode.h"
```

Comment:
Please don't use relative paths, use absolute paths instead i.e. instead of `../include/bencode.h` use `include/bencode.h`. Make can be configured to automatically manage these kinds of includes and these are more elegant.